blazor/datagrid/paging.md

```diff
@@ -329,6 +329,72 @@ In the below sample, we have customized the Pager component in DataGrid control
 
 > You can refer to our [Blazor Grid Pager Template](https://blazor.syncfusion.com/demos/datagrid/pager-template) online demo of Pager Template feature in Blazor DataGrid.
 
+## How to render Pager at the top of the Grid
+
+By default, Pager will be rendered at the bottom of the Grid. You can also render the Pager at the top of the Grid. This can be achieved by rendering the `SfPager` component.
```

Comment:
To render the Pager component at the top of the Grid, `SfPager` component is used.